### ISSUE TITLE
ktlint Confused by expected class

### ISSUE THREAD

**Issue by ScottPierce** [shyiko/ktlint#242]
Error:
```
ComponentContainerView.kt:7:37: Parameter should be on a separate line (unless all parameters can fit a single line) (parameter-list-wrapping)
ComponentContainerView.kt:7:79: Parameter should be on a separate line (unless all parameters can fit a single line) (parameter-list-wrapping)
```

Code:
```Kotlin
expect class ComponentContainerView {
  internal val componentContainers: MutableMap<Component<*>, SubContainerView>
}
```

Line 7 is the `internal val componentContainers` line.

It seems to be getting confused by the lack of constructor parenthesis, which is legal when you don't want to enforce a constructor on an expected class.

**Comment by shyiko** [comment]
```sh
$ cat ComponentContainerView.kt
expect class ComponentContainerView {
    internal val componentContainers: MutableMap<Component<*>, SubContainerView>
}

$ ktlint ComponentContainerView.kt --debug
[DEBUG] Discovered ruleset "standard"
[DEBUG] Discovered reporter "plain"
[DEBUG] Discovered reporter "json"
[DEBUG] Discovered reporter "checkstyle"
[DEBUG] Initializing "plain" reporter with {verbose=false, color=false}
[DEBUG] Checking /tmp/ComponentContainerView.kt
[DEBUG] 670ms / 1 file(s) / 0 error(s)

$ ktlint --version
0.24.0
```

Can you please double-check.

**Comment by ScottPierce** [comment]
I upgraded ktlint to 0.24.0. Not sure if I'm getting a different error message, but I found the issue was in the actual declaration.
